README.md

```diff
@@ -24,22 +24,139 @@ npm install
+quickly run `jiralint` without having to create any extra secrets, you can use
+your login credentials directly:
+
+```sh
+cli/dist/jiralint search -h [JIRA HOST] -j "project=MF order by created" \
+  --jira.username [USERNAME] --jira.password [PASSWORD]
+```
+
+### .jiralintrc
+
+To save time setting command-line parameters, you can configure `jiralint` using
+the `.jiralintrc` configuration file. This file will be resolved starting from
+your current working directory and searching up the file tree until a
+`.jiralintrc` file is (or isn’t) found.
+
+* Common Parameters:
+  * __jiraProtocol__: the protocol to use to connect to Jira - http or https
+  * __jiraHost__: the Jira hostname - e.g. jira.example.com
+  * __qualityFieldName__: the name of the field to store the quality in - customfield_12345
+  * __qualityReasonFieldName__: the name of the field to store the quality in - customfield_12346
+* OAuth connection:
+  * __jiraConsumerKey__: the name of the consumer secret you created in Jira
+  * __jiraConsumerSecret__: the consumer secret you created in Jira
+  * __accessToken__: the access token you crated with `jiralint auth`
+  * __accessSecret__: the access secret you crated with `jiralint auth`
+* Personal access token connection:
+  * __personalAccessToken__: the personal access token you created in Jira
+* User credentials connection:
+  * __username__: your jira username
+  * __password__: your jira password
+
+If you set all of the common parameters and a full set of connection parameters,
+you can now run jiralint search like this:
+
+```sh
+cli/dist/jiralint search -j "project=MF order by created"
+```
+
+#### Example OAuth `.jirilintrc`
+
+```json
+{
+  "jiraConsumerSecret": "LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCllvdXIgamlyYSBjb25zdW1lciBrZXkgd2lsbCBnbyBoZXJlCk1jRy9vU1Y1ajdQYjc0OHNjaEFMMEQ3NHlRbGZ6TDc0WXk5b3pnRUhxMFdnY1haM0tiQjVUcktjSk1zc2w2L2YKWC9lZzlQRmJORnY2WU0xZjNtSFBTVGM0TjZaY2JJM0szRHVvMm1YN0JYaVhUQVRXYXArL0dTN3RnUW5wNkpMYwo1RW1SbWM1cFJxTkE5cTBZOVZNWDh3d1dXWXNKRnMveGk0MCt6eXZFUHpZMCtEVnpIb0kxS0VQb0x5aGpPZXFYCkxTeFdZbnpUaXlwY0NucTdQaUJxT0FueW90QnJsb2lqakNsT1lLWjJ0QVJjaWRWdlN5VHZMR1ZNbk9NaE1YckcKbDZEUHFGOE1FUXNPMytXMTc2NHNBS1dMK25zNE5jRmpSL2U4L2NDN01XS01IZEdESS80cEd6bE5uOWZxV0tCMApEcDM0WkRqRTV5NXl1amRocmV4bGJyOEZDN2dPb0k4K2QwditaL0pCNjZUeVlCZzJvL0tuMjh3QkRoVHZweGZoCi9wQWxuTnVKZHpqeFlFZm91OVFOVENzL2F4TVNnUTF5TEVDZXdyTkY4bnduRHdPYkVPMndTcXM4U2FjNDZJUTMKL2pJU3hjNHZLT1lvT2dqRTZ5b0xCWjhSVUIvZXJGT2JUZUtHekRvM09sZzNaRVBWSjI1K2ErQmZsUXhIRFMxTApqR3lMSVVUZzQvdkVpQlUrelJnZlE0UVZPYzBYTElVcFhEcVFhOE9hSGx6YXJjYXJGMEZaWldnNXBhYklnVGRtCjZGbTFDNDhBS3lISzFDcG5ibU92NTV5d05QaWpINHhpVEh4RHZNbkVPQTRZaC9GU1c2a2lsMmZHQWMrUTZ6ZkEKaWx6bytqbWRza0lROUM3YkgzTmNlejhMbFRPUUxnVUFmRHNYTitndkxKSk1meUU1YUVmY0x6ZVJvMldlNyt3agpyNGpFUU1vSHFSVmtHOHVOMWlnQzljNERuMTJBbXF1bDhtaytuV3B0aXp4SDNuT016Z0xrNUhzOFlhNldwSldKCjdDRFV1M01vWnV4WVA1QlVySjJoZVBqOTY4dHVJUklRZ3F4UjlxaWt3WXdVemVBWXFZek9uUE10OWd5TXZ4MG0KLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQo=",
```

Comment:
I hope this is a made up key... maybe snip it in the middle for readability?

Reply:
Yep, it's made up. /dev/urandom. I made it about the length of the regular key so users would know that that's how big it was supposed to be, but happy to snip it.

Reply:
I like the idea of indicative length.